CONTRIBUTING.md

```diff
@@ -1,310 +1,31 @@
-
-```shell
-./risedev k  # shortcut for ./risedev kill
-./risedev clean-data
-```
-
-As RisingWave's codebase is constantly changing, and all persistent data might not be in a stable format, if there's
-some unexpected decode error, try `./risedev clean-data` first.
-
-### End-to-End Testing on CI
-
-Basically, CI is using the following two configuration to run the full e2e test suite:
-
-```shell
-./risedev dev ci-3cn-1fe
-./risedev dev ci-1cn-1fe
-```
-
-We may adjust the environment variable to enable some specific code to make all e2e tests pass. Refer to GitHub Action workflow for more information.
-
-## Developing Dashboard
-
-Currently, we have two versions of RisingWave dashboard. You may use RiseDev config to select which dashboard to use.
-
-Dashboard will be available at `http://127.0.0.1:5691/` on meta node.
-
-### Dashboard V1
-
-Dashboard V1 is a single HTML page. To preview and develop the web page, install Node.js, and
-
-```shell
-cd src/meta/src/dashboard && npx reload -b
-```
-
-Dashboard V1 is bundled by default along with meta node. Without any configuration, you may use the dashboard when
-the cluster is started.
-
-### Dashboard v2
-
-The development instructions for dashboard v2 is in [here](https://github.com/singularity-data/risingwave/blob/main/dashboard/README.md).
-
-## Observability
-
-RiseDev supports several observability components in RisingWave cluster.
-
-### Monitoring
-
-Uncomment `grafana` and `prometheus` services in `risedev.yml`, and you can view the metrics through a built-in Grafana dashboard.
-
-### Tracing
-
-Compute node supports streaming tracing. Tracing is not enabled by default, and you will need to
-use `./risedev configure` to enable tracing components. After that, simply uncomment `jaeger`
-service in `risedev.yml`.
-
-### Dashboard
-
-You may use RisingWave Dashboard to see actors in the system. It will be started along with meta node.
-
-### Logging
-
-The Rust components use `tokio-tracing` to handle both logging and tracing. The default log level is set as:
-
-* Third-party libraries: warn
-* Other libraries: debug
-
-If you need to adjust log levels, simply change the logging filters in `utils/logging/lib.rs`.
+If you have questions, please [create a Github issue](https://github.com/singularity-data/risingwave/issues/new/choose) or ask in the RisingWave Community channel on Slack. Please use the [invitation link](https://join.slack.com/t/risingwave-community/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw) to join the channel.
 
-## Misc Check
 
-For shell code, please run:
+## Table of contents
 
-```shell
-brew install shellcheck
-shellcheck <new file>
-```
+  - [Tests and miscellaneous checks](#misc-check)
+  - [Submit a PR](#submit-a-pr)
+    - [Pull Request title](#pull-request-title)
+    - [Pull Request description](#pull-request-description)
+    - [Sign the CLA](#sign-the-cla)
+  - [Update CI workflow](#update-ci-workflow)
```

Comment:
> Why update CI workflow, add new files, add new dependencies, check in PR from forks are part of CONTRIBUTING.md instead of developer docs?

Now I see the point. We can move these parts to developer guide, and keep only the content about submitting PR.